components/brave_wallet_ui/page/screens/onboarding/create-password/onboarding-create-password.tsx

```diff
@@ -51,6 +56,9 @@ export const OnboardingCreatePassword = (
   const isWalletCreated = useSelector(
     ({ wallet }: { wallet: WalletState }) => wallet.isWalletCreated
   )
+  const isCreatingWallet = useSelector(
+    ({ page }: { page: PageState }) => page.isCreatingWallet
```

Comment:
let's create a reusable selector function for this

Reply:
Done!